docs/user/contacts.md

```diff
@@ -0,0 +1,49 @@
+# Contacts
+
+## Table of Contents
+
+- [Bug Report](#Bug-Report)
+- [Proposal](#Proposal)
+- [Feature Request](#Feature-Request)
+- [Security Issue Report](#Security-Issue-Report)
+
+### Bug Report
+
+A bug is a demonstrable problem which produce incorrect result or to behave in unintended ways.
+Bug reports are helpful for developers who maintain the Vald project.
+
+A good bug report shouldn’t leave others needing to ask you for more information constantly.
+Please try to write as detail as possible in your bug report.
+
+Things to check before reporting a bug:
+- Check your environment
+Bug may occur depending on different version of Vald, Kubernetes and etc.
+
+- Identify the bug
+Developers need to reproduce the bug in the developers environment.
+So please Identify the bug and clarify how it occurs.
+Also please clarify the expected behavior too.
+
+- Use the Github issue search
+The bug may be reported before.
+So please check the Github issue before reporting.
+
+### Proposal
+
+The Vald project’s development process is design-driven. 
+The significant change to the library, or the architecture be discussed first.
```

Comment:
```suggestion
The significant change to the library, or the architecture will be discussed first.

```

Reply:
Thank you for your review.
I will fix it soon.